packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -965,14 +972,14 @@ class ImportedDatabaseCluster extends DatabaseClusterBase implements IDatabaseCl
   public readonly clusterIdentifier: string;
   public readonly connections: ec2.Connections;
   public readonly engine?: IClusterEngine;
-  public readonly secret?: secretsmanager.ISecret;
 
   private readonly _clusterResourceIdentifier?: string;
   private readonly _clusterEndpoint?: Endpoint;
   private readonly _clusterReadEndpoint?: Endpoint;
   private readonly _instanceIdentifiers?: string[];
   private readonly _instanceEndpoints?: Endpoint[];
 
+  public readonly secret?: secretsmanager.ISecret;
   protected readonly enableDataApi = false;
```

Comment:
I wonder why `enableDataApi` is `true` for `ImportedServerlessCluster` and `false` here 🤔

Reply:
@lpizzinidev ServerlessCLuster is only supporting Aurora SQL construct v1, which mean dapa api is always supported. In that case having it at True don't create issues.
For Aurora Serverless V2, we use DatabaseCluster construct which is also used for other type of RDS DB than aurora, which mean it is not always supported. In that case having it default at false is better to avoid enabling it on non supported DB type.

I do not recommend changing those default values.